apsuite/commisslib/rfvoltage_calibration.py

```diff
@@ -82,7 +91,7 @@ def calc_voltage_span(self):
     def set_bbb_drive_frequency(self, sync_freq):
         """."""
         harm_nr = 864
-        rf_freq = self.devices['rfcav'].dev_rfgen.frequency
+        rf_freq = self.devices['rfcavs'][0].dev_rfgen.frequency
```

Comment:
Change this method to use the revolution frequency provided by the BbB. This way it is not necessary to use a frequency offset.

Reply:
Done. Thanks